### PR TITLE
Add doc to replace dcos overlay with calico network

### PR DESCRIPTION
**Pull request by mainred**
## High-level description

What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (required)

  - [https://jira.d2iq.com/browse/D2IQ-62521](https://jira.d2iq.com/browse/c) add doc to replace DC/OS overlay with Calico

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by mainred** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by mainred** [comment]
@mesosphere-mergebot request-review  @wavesoft @vespian

**Comment by orlandohohmeier** [comment]
@dcos/docs can you help with a style/language review?

**Comment by JohnB64-JBV** [comment]
Approved with my changes noted above.

**Comment by mainred** [comment]
@JohnB64-JBV , awesome suggestion from a technical writer's view. It would be better if you could give review comments instead.

File: packages/calico/README.md [hunk 1]
```diff
@@ -4,30 +4,34 @@ This package provides DC/OS Calico component to support Calico networking contai
 
 <!-- MarkdownTOC -->
 
-* [1. System Requirements](#1-system-requirements)
```
Comment (mainred) [inline packages/calico/README.md]: Will adjust the change of TOC later.

**Comment by DominikDary** [comment]
Closing this in favor of https://github.com/dcos/dcos/pull/7023 which incorporated all the comments from John.

**Pull request closed without merging**